AutoscoperM/AutoscoperM.py

```diff
@@ -110,9 +110,9 @@ def registerAutoscoperSampleData(dataType, version, checksum):
 def sampleDataConfigFile(dataType):
     """Return the trial config filename."""
     return {
-        "2023-08-01-Wrist": "2023-07-20-Wrist.cfg",
-        "2023-08-01-Knee": "2023-07-26-Knee.cfg",
-        "2023-08-01-Ankle": "2023-07-20-Ankle.cfg",
+        "2025-01-12-Wrist": "2023-07-20-Wrist.cfg",
+        "2025-02-10-Knee": "Knee_Sample_Data.cfg",
```

Comment:
The original idea was to have the name of the configuration file match the sample data archive where it was coming from. That said, standardizing on `<Name>_Sample_Data.cfg` is sensible.